db/migrations/20170915150152_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.raw('UPDATE movies SET name = title');
```

Comment:
I believe this is going to overwrite all the newly added movies which will have a `null` `title` value. You're going to want to have a `where` clause that checks that the `title` value is worth writing to `name`. 

So the change is either going to be here or we need to be double writing in the application instead of deleting the field.